### DISCUSSION TITLE
Theme & Localization

### DISCUSSION THREAD

**Discussion by MaxGasta** [OrchardCMS/OrchardCore#7015] [Q&A]
Hello.
is there a way to bind some .PO files to a specific theme?

If I put the files in the  "CMS/Localization" everything works well, but those files are theme's specific.
I would like to put them in the myTheme's project and managed them by a recipe.

Any suggestion?  

Thanks for help

**Comment by hishamco** [comment]
You could extend the localization APIs by implementing your own `ILocalizationFileLocationProvider` to load the localization resources from your theme folder
